app_base.htm

```diff
@@ -0,0 +1,58 @@
+<%inherit file="/homepage/templates/base.htm" />
+<%! from catalog import models as cmod %>
+
+<%block name="navitems">
```

Comment:
I would indent everything within this block to make it more readable

Reply:
oh good point, ill do that.